modules/cli/src/integrationTest/java/org/apache/ignite/internal/cli/commands/questions/ItConnectToClusterTest.java

```diff
@@ -132,6 +138,33 @@ void connectOnStartAndSave() throws IOException {
                 .isEqualTo("http://localhost:10300");
     }
 
+    @Test
+    @DisplayName("Should ask to connect to different URL")
+    void connectToAnotherUrl() throws IOException {
+        // Given prompt before connect
+        String promptBefore = Ansi.OFF.string(promptProvider.getPrompt());
+        assertThat(promptBefore).isEqualTo("[disconnected]> ");
+
+        // And connected
+        execute("connect");
+
+        // And answer is "y"
+        bindAnswers("y");
+
+        // When connect to different URL
+        execute("connect", "http://localhost:10301");
+
+        // Then
+        assertAll(
+                this::assertErrOutputIsEmpty,
+                () -> assertOutputIs("Connected to http://localhost:10300" + System.lineSeparator()
+                + "Connected to http://localhost:10301" + System.lineSeparator())
```

Comment:
That looks weird... Say I am connected to http://localhost:10300 and want to switch to another node and type `connect http://localhost:10301` and see 

```
Connected to http://localhost:10300
Connected to http://localhost:10301
```

It looks like a bug for the user. I suggest dropping the first line.